e2e/e2e_test.go

```diff
@@ -330,23 +342,18 @@ func generateNetAttachDefSpec(name, namespace, config string) *nettypes.NetworkA
 func macvlanNetworkWithWhereaboutsIPAMNetwork() *nettypes.NetworkAttachmentDefinition {
 	macvlanConfig := `{
         "cniVersion": "0.3.0",
-      	"disableCheck": true,
-        "plugins": [
-            {
-                "type": "macvlan",
-              	"master": "eth0",
-              	"mode": "bridge",
-              	"ipam": {
-                    "type": "whereabouts",
-                    "leader_lease_duration": 1500,
-                    "leader_renew_deadline": 1000,
-                    "leader_retry_period": 500,
-                    "range": "10.10.0.0/16",
-                    "log_level": "debug",
-                    "log_file": "/tmp/wb"
-              	}
-            }
-        ]
+		"type": "macvlan",
+		"master": "eth0",
+		"mode": "bridge",
+		"ipam": {
+			"type": "whereabouts",
+			"leader_lease_duration": 1500,
+			"leader_renew_deadline": 1000,
+			"leader_retry_period": 500,
+			"range": "10.10.0.0/16",
+			"log_level": "debug",
+			"log_file": "/tmp/wb"
+		}
```

Comment:
https://github.com/k8snetworkplumbingwg/whereabouts/pull/185/commits/4f2a6911270b850e02f356bbf17bc9543ba15285

I was wrong in adapting this in this PR, it is not needed, since whereabouts is only receiving the configuration for a single plugin at a time. On the other hand, it is required for the ip reconciliation loop.

If you try the test I've added in that commit without the associated code changes, you'll see it fails.